src/main/java/com/mcmoddev/lib/data/MaterialNames.java

```diff
@@ -0,0 +1,44 @@
+package com.mcmoddev.lib.data;
```

Comment:
This is part of BMe, not MMDlib, if it's going to exist at all in the lib, it should only contain names of materials in vanilla

Reply:
I agree...

but... I moved this here to get rid of 20+ compile errors for tests against material names.

I say, let's get this to work and make an issue so we don't forget about refactoring this part.

Reply:
kk

Reply:
all non-vanilla materials removed.

Reply:
Including several vanilla materials being removed too:
diamond, emerald, gold, obsidian, prismarine, redstone, stone